pkg/services/ngalert/migration/persist.go

```diff
@@ -0,0 +1,182 @@
+package migration
+
+import (
+	"context"
+	"fmt"
+
+	alertingNotify "github.com/grafana/alerting/notify"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	apiModels "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
+	migmodels "github.com/grafana/grafana/pkg/services/ngalert/migration/models"
+	migrationStore "github.com/grafana/grafana/pkg/services/ngalert/migration/store"
+	"github.com/grafana/grafana/pkg/services/ngalert/models"
+)
+
+// Sync is a helper struct for persisting migration changes to the database.
+type Sync struct {
```

Comment:
do we need to make it public?

Reply:
No, it doesn't need to be public right now. Holdover from a different version where it was in its own package. I'll remove it for now.